wagtail_wordpress_import/test/tests/test_shortcode_handlers.py

```diff
@@ -231,21 +231,22 @@ class CaptionHandler(BlockShortcodeHandler):
 
 
 class TestShortcodeHandlerRegistration(TestCase):
-    def test_caption_handlers_are_registered(self):
-        @register("caption")
-        class CaptionHandler(BlockShortcodeHandler):
-            shortcode_name = "caption"
+    def test_included_shortcode_handlers_are_registered(self):
+        # prime the SHORTCODE_HANDLERS
+        # note this class has not been registered
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
 
-        @register("form")
-        class FormHandler(BlockShortcodeHandler):
-            shortcode_name = "form"
+        registered_handlers = SHORTCODE_HANDLERS.keys()
+        self.assertIn("caption", registered_handlers)
```

Comment:
**Suggestion**: We're declaring class FooHandler, but checking that "caption" is in the registered handlers. I'm not sure what the intent of this test is. Did you mean to include:

```suggestion
        self.assertIn("caption", registered_handlers)
        self.assertNotIn("foo", registered_handlers)
```

?

Reply:
Hmm, That first test was bonkers I think and really not needed as other earlier tests would fail if it wasn't registered. I updated the remaining test to check for both keys.

**Question** Do you think thats OK?

Reply:
I still think there's merit in a test that something isn't there, and then testing that it is there.

Reply:
Thanks Nick, I added a second test.